File: platforms/ios/Dezel/Dezel/Modules/Dialog/DialogModule.swift
/**
 * @class DialogModule
 * @since 0.1.0
 * @hidden
 */
open class DialogModule: Module {

	//--------------------------------------------------------------------------
	// MARK: Methods
	//--------------------------------------------------------------------------

	/**
	 * @inherited
	 * @method initialize
	 * @since 0.1.0
	 */
	override open func initialize() {
		self.context.registerClass("dezel.dialog.Alert", value: Alert.self)
		self.context.registerClass("dezel.dialog.AlertButton", value: AlertButton.self)
	}
}

File: platforms/android/Dezel/dezel/src/main/java/ca/logaritm/dezel/modules/dialog/DialogModule.kt
package ca.logaritm.dezel.modules.dialog

import ca.logaritm.dezel.core.JavaScriptContext
import ca.logaritm.dezel.core.Module

/**
 * @class DialogModule
 * @since 0.1.0
 */
open class DialogModule(context: JavaScriptContext): Module(context) {

	//--------------------------------------------------------------------------
	// Methods
	//--------------------------------------------------------------------------

	/**
	 * @inherited
	 * @method initialize
	 * @since 0.1.0
	 */
	override fun initialize() {
		this.context.registerClass("dezel.dialog.Alert", JavaScriptAlert::class.java)
		this.context.registerClass("dezel.dialog.AlertButton", JavaScriptAlertButton::class.java)
	}
}

